crates/bevy_render/src/color/conversions.rs

```diff
@@ -0,0 +1,231 @@
+use super::*;
+
+trait SrgbColorSpace {
+    fn linear_to_nonlinear_srgb(self) -> Self;
+    fn nonlinear_to_linear_srgb(self) -> Self;
+}
+
+// source: https://entropymine.com/imageworsener/srgbformula/
+impl SrgbColorSpace for f32 {
+    fn linear_to_nonlinear_srgb(self) -> f32 {
+        if self <= 0.0 {
+            return self;
+        }
+
+        if self <= 0.0031308 {
```

Comment:
For floating-point equality checks, should we be using some sort of `epsilon` or a different crate like `float-cmp`?